src/core/mesh/qgsmeshtimesettings.h

```diff
@@ -53,47 +53,16 @@ class CORE_EXPORT QgsMeshTimeSettings
     };
 
     QgsMeshTimeSettings();
-    //! Constructs relative time format settings with defined offset in hours
-    QgsMeshTimeSettings( double relativeTimeOffsetHours, const QString &relativeTimeFormat );
-    //! Constructs absolute time format settings with defined reference time
-    QgsMeshTimeSettings( const QDateTime &absoluteTimeReferenceTime, const QString &absoluteTimeFormat );
 
     //! Writes configuration to a new DOM element
     QDomElement writeXml( QDomDocument &doc, const QgsReadWriteContext &context ) const;
     //! Reads configuration from the given DOM element
     void readXml( const QDomElement &elem, const QgsReadWriteContext &context );
 
-    //! Returns whether to use absolute time format
-    bool useAbsoluteTime() const;
-    //! Sets use absolute time flag
-    void setUseAbsoluteTime( bool useAbsoluteTime );
-
-    //! Returns number of offset hours for relative time formatting
-    double relativeTimeOffsetHours() const;
-    //! Sets number of offset hours for relative time formatting
-    void setRelativeTimeOffsetHours( double relativeTimeOffsetHours );
-
-    /**
-     * Returns number of seconds used as interval for dataset playback
-     * \since QGIS 3.12
-     */
-    double datasetPlaybackInterval() const; SIP_SKIP
-
-    /**
-     * Sets number of seconds used as interval for dataset playback
-     * \since QGIS 3.12
-     */
-    void setDatasetPlaybackInterval( double seconds ); SIP_SKIP
-
     //! Returns format used for relative time
-    QString relativeTimeFormat() const;
+    QString relativeTimeFormat() const; //not necessary with QGIS time setting
```

Comment:
I do not understand comment "//not necessary with QGIS time setting" also can you please move comment to the doxy //! comment above?